README.md

```diff
@@ -9,26 +9,25 @@ Read more [here](http://datafordemocracy.org/projects/ethics.html).
 ## Ethics Principles
 ---
 <ul>
-    <li>Its my job to understand, mitigate and communicate the presence of bias in algorithms.</li>
-    <li>Be responsible for maximizing social benefit and minimizing harm.</li>
-    <li>Practice humility and openness.</li>
+    <li>Bias will exist. It's my responsibility to understand, mitigate and communicate the presence of bias in algorithms.</li>
+    <li>It's my responsibility to increase social benefit while minimizing harm.</li>
+    <li>I will practice humility and openness.</li>
     <li>I will know my data and help future users know it as well.</li>
-    <li>Make reasonable efforts to know and document its origins and document its transformation.</li>
-    <li>Bias will exist. Measure it. Plan for it.</li>
-    <li>Thou shalt document transparently, accessibly, responsibly,  reproducibly, and communicate.</li>
```

Comment:
"Thou shalt" throws off the tone of the document. If moving to "pledge" instead of "directive," it would need to be removed. But also, this document should avoid channeling the tone of any specific religious texts.